server/src/main/java/org/opensearch/node/Node.java

```diff
@@ -1155,6 +1155,7 @@ public Node start() throws NodeValidationException {
             : "clusterService has a different local node than the factory provided";
         transportService.acceptIncomingRequests();
         extensionsOrchestrator.extensionsInitialize();
+        extensionsOrchestrator.getNamedWriteables();
```

Comment:
How are these namedWritables registered with OpenSearch NamedWritableRegistry?

Reply:
Currently this is a placeholder trigger. Registering extensions named writeables within the central named writeable registry is a larger problem that I have linked [this issue](https://github.com/opensearch-project/OpenSearch/issues/4067) to.